### DISCUSSION TITLE
E0D05D8D-1224-4DA6-933E-94356A4C6214

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#146] [General]
# E0D05D8D-1224-4DA6-933E-94356A4C6214

HD 164922, aka Gliese 700.2, is a G-type main sequence star located about 71.81 lightyears from the solar system. the star is joined by an entourage of planets: a mini-Neptune (d= 4.767 Me) two Neptune mass planets (e= 10.52 Me | c= 14.302 Me) and a greater-Saturn mass planet (b= 109.965 Me). the inner most planet (d) has a slightly eccentric orbit.

https://celestia.mobi/addon?item=E0D05D8D-1224-4DA6-933E-94356A4C6214

**Comment by exocritic5457** [comment]
This addon does add the planets around HD 164922, but it also leaves the same planets, labeled as Gliese 700.2's, in, though invisible at that
